misk/src/test/kotlin/misk/web/interceptors/RequestLoggingInterceptorTest.kt

```diff
@@ -133,6 +140,19 @@ internal class RequestLoggingInterceptorTest {
     return httpClient.newCall(request.build()).execute()
   }
 
+  @Test
+  fun capturesSubsetOfHeaders() {
+    assertThat(webTestClient.call("/call/withHeaders") {
+      post("hello".toRequestBody(MediaTypes.APPLICATION_JSON_MEDIA_TYPE))
+      addHeader(FakeCallerAuthenticator.SERVICE_HEADER, "some-value")
+    }.response.isSuccessful)
+      .isTrue()
+    val messages = logCollector.takeMessages(RequestLoggingInterceptor::class)
+    assertThat(messages).containsExactly(
```

Comment:
Because we're wanting to ensure non-approved headers aren't in the message, maybe add an assertion that those header(s) are NOT in the message explicitly?

Reply:
That's fair, will add it to the test.